app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
```

Comment:
Not exactly, but I see what you are saying. It's a non-issue for the moment because we decided to get rid of the picture there to allow for more room to list our available games. This will be in the PR to follow.